cmd/kubeadm/app/cmd/token.go

```diff
@@ -213,8 +217,14 @@ func RunCreateToken(out io.Writer, client clientset.Interface, token string, tok
 		}
 	}
 
+	for _, group := range extraGroups {
```

Comment:
+1, it's just a client side check but it's good validation so we can fail fast with a nice error message.